nuxeo-core/nuxeo-core-bulk/src/test/java/org/nuxeo/ecm/core/bulk/DummyLoginAs.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * (C) Copyright 2012-2013 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.nuxeo.ecm.core.bulk;
+
+import java.security.Principal;
+
+import javax.security.auth.login.LoginContext;
+import javax.security.auth.login.LoginException;
+
+import org.nuxeo.ecm.core.api.impl.UserPrincipal;
+import org.nuxeo.ecm.core.api.local.ClientLoginModule;
+import org.nuxeo.runtime.api.login.LoginAs;
+
+/**
+ * @since 10.3
```

Comment:
Describe the purpose, I am not sure to understand why such basic test class must be defined in bulk core